Backend/hunit-tests/package.yaml

```
@@ -0,0 +1,94 @@
+- TupleSections
+- TypeApplications
+- TypeFamilies
+- TypeOperators
+- ViewPatterns
+
+library:
+  source-dirs: src
+  exposed-modules:
+  - Dashboard
+  other-modules:
+    - RegistrationUnitTests
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)